src/lib/components/accounts/Unverified.svelte

```diff
@@ -0,0 +1,65 @@
+<!-- @component
+This component displays anywhere we need to explain verification to
+a user who is logged in but has `verified_journalist = false`.
```

Comment:
I think we want this component to rendering conditional on this rule to keep things DRY. If we ever change the heuristic for when to show the message, we'll change it in one place instead of multiple files. It also means that if I want to include the Unverified tip in another component, I can trust it will appear appropriately without extra work.

We could also split the difference by making the check an optional function prop with the signature `(user: User) => boolean` and a default implementation.

Reply:
For the upload view, we're displaying this in place of the normal header, so we need that check anyway.